example/main.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"github.com/kataras/golog"
+	"github.com/vitwit/go-fass/openfass-go"
```

Comment:
Can you the file structure in a way to import just
`github.com/vitwit/go-fass` instead of this

Reply:
done anil.